modules/protoc-gen-validate/1.0.4/patches/pgv.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/python/BUILD b/python/BUILD
```

Comment:
What is the source of these patches? Some of them look functional rather than related to Bzlmod